streams/src/test/java/org/apache/kafka/streams/integration/IQv2StoreIntegrationTest.java

```diff
@@ -816,29 +816,65 @@ private <T> void shouldHandleRangeQueries(final Function<T, Integer> extractor)
         shouldHandleRangeQuery(
             Optional.of(1),
             Optional.of(3),
+            true,
             extractor,
-            mkSet(1, 2, 3)
+            Arrays.asList(2, 1, 3)
 
         );
         shouldHandleRangeQuery(
             Optional.of(1),
             Optional.empty(),
+            true,
             extractor,
-            mkSet(1, 2, 3)
+            Arrays.asList(2, 1, 3)
 
         );
         shouldHandleRangeQuery(
             Optional.empty(),
             Optional.of(1),
+            true,
             extractor,
-            mkSet(0, 1)
+            Arrays.asList(0, 1)
 
         );
         shouldHandleRangeQuery(
             Optional.empty(),
             Optional.empty(),
+            true,
             extractor,
-            mkSet(0, 1, 2, 3)
+            Arrays.asList(0, 2, 1, 3)
+
```

Comment:
Seems the formatting is off here... the empty line should be after the close `);`

```
    Arrays.asList(0, 2, 1, 3)
);

shouldHandleRangeQuery(
```

It's already messy in the existing code -- can you clean it up everywhere?

Reply:
Yes, that look wried, I have already removed extra lines.